trunk/wordpress-sparkpost.php

```diff
@@ -28,9 +28,9 @@
 }
 $sp = new SparkPost();
 
-if ($sp->get_option('enable_sparkpost')) {
+if ($sp::get_option('enable_sparkpost')) {
```

Comment:
I am even wonder how it works! Seems I can call static method through instance which is weird.